APP/Pages/Meetup.razor

```diff
@@ -1,39 +1,42 @@
 ﻿@using APP.Data.Models;
 @using APP.Interfaces;
+@using Microsoft.Extensions.Configuration;
 @using APP.Services;
+@inject IConfiguration Configuration;
 @inject LoggedUserState loggedUserState
 
```

Comment:
Requires the
@using APP.States;
@using System.ComponentModel;
namespaces to access the loggeduserState and propertyChanged events.

Reply:
added